src/worker/keyman-api-service.ts

```diff
@@ -0,0 +1,36 @@
+import { keyboardDataObj, KeymanObj } from "./models";
+
+export class KeymanApi {
+  /**
+   * API configs
+   */
+  baseUrl = "https://api.keyman.com/search";
+  /**
+   * Query data to searche only for languages that match
+   */
+  params = { q: "l" };
+  url: URL;
+  langageArray: Array<keyboardDataObj>;
```

Comment:
Please use this notation instead:

```suggestion
  langageArray: keyboardDataObj[];
```

Although, I do suggest using a different name for this interface!